pages/windows/msg.md

```diff
@@ -0,0 +1,19 @@
+# msg
+
+> Send a message to a specific user or session.
+
+- Send a message to a specified user or session:
+
+`msg {{username|session_name|session_id}} {{message}}`
+
+- Send a message from stdin:
+
+`{{message}} | msg`
```

Comment:
Shouldn't it be echo {{message}} or whatever equivalent of that is in windows ? Simply typing "message" works ?